### ISSUE TITLE
Some details I didn`t work out

### ISSUE THREAD

**Issue by PYM-242115005** [Monu-Khicher-1/multi-stage-learning#3]
Hi,Im a student who wants to reproduce the experiment.After reading the file README,I wonder if did it wrong.I just built a virtual  environment satisfying the need,and cut the Celeb-V2 vedios into images 1fps,cropped all images,then masked the training data. After this,I started the three-stage training as the README said.But the result is definitely wrong.I actually wonder what should I adjust more as a fresh one to reproduce it.By the way,I found a minority of images were not masked when I tried to do so.I just deleted them.

Moreover,there`re some issues.The first is what the simple data refers to,is it the sample_train_data or the data without being masked?
The second is if I don`t need to do anymore to train,just to process the data and set the hyperparameters.And the last one is ,in test.py,it maintain a function load_data from dataset.test_loader,but the file test_loader.py doesn`t exist,is it just loader.py or something else?

By the way ,I just did as the 3 stage did,data in the second stage is cropped data,but in the end,the training acc is just about 0.7,and the validating acc is even about 0.35.They both haven`t grown for dozens of epoches.

I`m so confused now,and I really need a instruction to get a result as excellent as yours based on the codes you provided on github! Hoping for your reply.Thanks!!!
![loss](https://github.com/user-attachments/assets/10f0fc59-6de7-457c-99a4-fc852eb8b949)
![performance](https://github.com/user-attachments/assets/d246de92-fe10-4397-bfdf-8e23ea7a6672)
![roc_curve](https://github.com/user-attachments/assets/97deed9f-0368-4bb2-96aa-c607ef4f92bd)
